cypress/e2e/pages/topicPage/index.cy.js

```diff
@@ -1,30 +1,71 @@
-import config from '../../../support/config/services';
-import getPaths from '../../../support/helpers/getPaths';
-import serviceHasPageType from '../../../support/helpers/serviceHasPageType';
-
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
 import crossPlatformTests from './tests';
-import visitPage from '../../../support/helpers/visitPage';
-import { getTopicPagePath } from './helpers';
+import { TOPIC_PAGE } from '../../../../src/app/routes/utils/pageTypes';
+
+const topicPagesToTest = [
+  {
+    path: '/arabic/topics/cwr9j7nv58nt',
+    service: 'arabic',
+    runforEnv: ['local', 'test', 'live'],
+    pageType: TOPIC_PAGE,
+    tests: [crossPlatformTests],
```

Comment:
This (and all other references) can now be replaced with the following, as we have declared the tests above.

 ```suggestion
     tests,
```